akka-projections-core/src/main/scala/akka/projections/Projections.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projections
+
+import akka.Done
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Sink, Source}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object Projections {
```

Comment:
May I suggest that `Projections` is really a Sink?

Perhaps something like: `mySource.to(projection.asSink(commitStrategy))`

Reply:
You mean, have a method in the projection that transforms it into a `Sink`?
Yeah, I think so. That can be interesting.

Reply:
@renatocaval Yes, I think the fewer distinct concept for composing things, the better. If it is "just a sink" then it will seem familiar.

Reply:
@viktorklang, I'm starting to work on this stuff again, but I'm closing this PR in favour of that other one https://github.com/akka/akka-projection/pull/5. 

Regarding the usage of Sink here, I think this will become obsolete. Let's discuss it on #5.